### DISCUSSION TITLE
How can I set default Zoom size

### DISCUSSION THREAD

**Discussion by XomaDev** [patrykandpatrick/vico#347] [Questions] [answered]
How do I set a default Zoom size for the graph?
Because each time the user would require to manually zoom out to see the whole data.

I've searched but couldnt find anything that would do it.
Thanks

**Comment by XomaDev** [comment]
Would it be possible to configure the chart in such a way that it cans see large sum of data at once or the whole data?

**Comment by patrickmichalik** [comment, marked as answer]
Hello! The amount of data that’s initially visible depends on the dimensions and spacing of the `Chart`’s elements. For example, for a `ColumnChart`, decreasing the width and spacing of the columns will make more of them fit in the viewport. Also, if you turn off scrolling, the `Chart` will be automatically scaled such that all entries are visible. We plan on making it possible to apply such automatic scaling to scrollable charts too, which will make it possible to zoom in and scroll around.

**Reply by ahmed2146** [reply]
Hi, have you added this thing? I don't want to give the user a scaling option, but I have to display all 15 entries at the same time

**Reply by patrickmichalik** [reply]
These changes aren’t available yet, @ahmed2146, but if you don’t need your chart to be zoomable, you can disable scrolling, and the content will be scaled down to fit the width of the viewport.

**Reply by ahmed2146** [reply]
how can i disable scrolling ?

**Reply by patrickmichalik** [reply]
Via `ChartScrollSpec`. Are you using Vico with Jetpack Compose or the view system?

**Reply by ahmed2146** [reply]
using jetpack compose

**Reply by patrickmichalik** [reply]
If so, please use `Chart(..., chartScrollSpec = rememberChartScrollSpec(isScrollEnabled = false))`.

**Reply by ahmed2146** [reply]
thanks

**Reply by patrickmichalik** [reply]
You’re welcome!
